CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unrelased
+### Added
+- Note for recent pending comments in partial diffs ([#169](https://github.com/scm-manager/scm-review-plugin/pull/169))
+
```

Comment:
Conflict because of this. Please start creating changelog entry files in all plugins.